api/rest/restcore/projects_rest.php

```diff
@@ -39,6 +39,14 @@
 	# Project versions
 	$g_app->post( '/{id}/versions', 'rest_project_version_add' );
 	$g_app->post( '/{id}/versions/', 'rest_project_version_add' );
+
+	# Project subprojects
+	$g_app->post( '/{id}/subprojects', 'rest_project_subproject_add' );
+	$g_app->post( '/{id}/subprojects/', 'rest_project_subproject_add' );
+	$g_app->patch( '/{id}/subprojects/{subproject_id}', 'rest_project_subproject_update' );
```

Comment:
I wonder if we should make subproject_id optional in the patch since it should be specified in the body along with inherit_parent using subproject id or subproject name under the project field.